database/redis/latest.Dockerfile

```
@@ -0,0 +1,11 @@
+# https://github.com/redis/docker-library-redis/blob/e5650da99bb377b2ed4f9f1ef993ff24729b1c16/7.4/debian/Dockerfile
+# https://hub.docker.com/_/redis/tags
+FROM redis:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)